net/grpc/gateway/docker/protoc_plugin/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM grpcweb/prereqs
+FROM grpcweb/prereqs:latest
```

Comment:
not sure if this change would work with local builds with `docker compose`..

Reply:
You're right, this is a hack and it still doesn't work right, I'll leave a longer comment in the conversation flow